app/datatables/custom_datatable_v2.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+class CustomDatatableV2
```

Comment:
We should get rid of references to "datatable" and just have generic "table services".